resources/dicts/thoughts/alive/apprentice.json

```diff
@@ -3,38 +3,38 @@
         "id": "gen_warrior_app",
         "thoughts":[
             "Feels worn out from training",
-            "Wonders if they'll be allowed to go night-hunting soon",
+            "Wonders if {PRONOUN/m_c/subject}'ll be allowed to go night-hunting soon",
             "Is resting in the sun after a long day of training",
-            "Wonders if they'll have to fight a battle anytime soon",
+            "Wonders if {PRONOUN/m_c/subject}'ll have to fight a battle anytime soon",
             "Bats at a beetle that wandered into camp",
-            "Is saving the softest bedding for their own nest",
-            "Practices their newest battle moves",
-            "Hopes they'll learn something new today",
-            "Is feeling proud of the progress they've made",
-            "Practices their hunting on a smooth rock they found",
+            "Is saving the softest bedding for {PRONOUN/m_c/poss} own nest",
+            "Practices {PRONOUN/m_c/poss} newest battle moves",
+            "Hopes {PRONOUN/m_c/subject}'ll learn something new today",
+            "Is feeling proud of the progress {PRONOUN/cat/subject}{VERB/cat/'ve/'s} made",
+            "Practices {PRONOUN/cat/object} hunting on a smooth rock {PRONOUN/m_c/subject} found",
```

Comment:
The first pronoun tag in this line should be ` {PRONOUN/m_c/poss} `

Reply:
Ah thankyou, I had a feeling I missed some.